InnoGotchi.API/Infrastructure.Services/AvatarService.cs

```diff
@@ -0,0 +1,29 @@
+﻿using InnoGotchi.Application.Contracts.Services;
+using InnoGotchi.Application.DataTransferObjects.User;
+
+namespace Infrastructure.Services
+{
+    public class AvatarService : IAvatarService
+    {
+        public async Task<string> CreateImageAsync(Guid userId, AvatarChangingDto avatarDto)
+        {
+            var imageName = $"{userId}-{avatarDto.FileName}";
+            var path = Path.Combine(Directory.GetCurrentDirectory(), "wwwroot", "images", "avatars", imageName);
+            var bytes = Convert.FromBase64String(avatarDto.Base64Image);
+
+            using var stream = File.Create(path);
+            stream.Write(bytes);
+
+            return imageName;
+        }
+
+        public void DeleteOldImage(string avatarPath)
+        {
+            if (avatarPath != "default.jpg")
```

Comment:
Do not use literal values, either numeric or strings, in your code other than to define symbolic constants

Reply:
fixed